corehq/sql_db/routers.py

```diff
@@ -45,6 +46,8 @@ def allow_migrate(db, app_label):
     if app_label == ICDS_REPORTS_APP:
         db_alias = get_icds_ucr_db_alias()
         return db_alias and db_alias == db
+    elif app_label == SYNCLOGS_APP:
+        return hasattr(settings, "SYNCLOGS_SQL_DB_ALIAS") and db == settings.SYNCLOGS_SQL_DB_ALIAS
```

Comment:
`settings` will always have `SYNCLOGS_SQL_DB_ALIAS` so the `hasattr` check isn't necessary

Reply:
Thanks, fixed.